test/run_norm_00.sh

```diff
@@ -1,26 +1,29 @@
 WORKSPACE=/tmp
 RUNDIR=../starkjs
```

Comment:
here we can define CUR_DIR=$(cd $(dirname $0);pwd), then we can get the abs directory of RUNDIR, thus we can start this script anywhere.